python/tests/aggregate_tests6/table.py

```diff
@@ -0,0 +1,305 @@
+#     EXTRACT(EPOCH FROM f_c6) AS m_c6_seconds
+# FROM agg_view;
+
+
+class aggtst_binary_table(TstTable):
+    """Define the table used by the binary tests"""
+
+    def __init__(self):
+        self.sql = """CREATE TABLE binary_tbl(
+                      id INT,
+                      c1 BINARY(4),
+                      c2 BINARY(4) NULL)WITH ('append_only' = 'true')"""
+        self.data = [
+            {"id": 0, "c1": [12, 22, 32], "c2": None},
+            {"id": 0, "c1": [23, 56, 33, 21], "c2": [55, 66, 77, 88]},
+            {"id": 1, "c1": [23, 56, 33, 21], "c2": [99, 20, 31, 77]},
+            {"id": 1, "c1": [49, 43, 84, 29], "c2": [32, 34, 22, 12]},
+        ]
+
+        # Equivalent SQL for Postgres
+
+        # CREATE TABLE binary_tbl (
+        #     id INT,
+        #     c1 BYTEA,
+        #     c2 BYTEA NULL
+        # );
+
+        # INSERT INTO binary_tbl (id, c1, c2) VALUES
+        #     (0, '\x0c1620', NULL),
+        #     (0, '\x17382115', '\x37424d58'),
+        #     (1, '\x17382115', '\x63141f4d'),
+        #     (1, '\x312b541d', '\x2022160c');
+
+
+class aggtst_unsigned_int_table(TstTable):
+    def __init__(self):
+        self.sql = """CREATE TABLE un_int_tbl(
+                      id INT NOT NULL,
+                      c1 TINYINT UNSIGNED,
+                      c2 TINYINT UNSIGNED NOT NULL,
+                      c3 SMALLINT UNSIGNED,
+                      c4 SMALLINT UNSIGNED NOT NULL,
+                      c5 INT UNSIGNED,
+                      c6 INT UNSIGNED NOT NULL,
+                      c7 BIGINT UNSIGNED,
+                      c8 BIGINT UNSIGNED NOT NULL)WITH ('append_only' = 'true')"""
+
+        self.data = [
+            {
+                "id": 0,
+                "c1": 72,
+                "c2": 64,
+                "c3": None,
+                "c4": 16002,
+                "c5": 781245123,
+                "c6": 651238977,
+                "c7": None,
+                "c8": 284792878783,
+            },
+            {
+                "id": 1,
+                "c1": 61,
+                "c2": 64,
+                "c3": 14257,
+                "c4": 15342,
+                "c5": 963218731,
+                "c6": 749321014,
+                "c7": 367192837461,
+                "c8": 265928374652,
+            },
+            {
+                "id": 0,
+                "c1": None,
+                "c2": 45,
+                "c3": 13450,
+                "c4": 14123,
+                "c5": 812347981,
+                "c6": 698123417,
+                "c7": 419283746512,
+                "c8": 283746512983,
+            },
+            {
+                "id": 1,
+                "c1": None,
+                "c2": 48,
+                "c3": 12876,
+                "c4": 13532,
+                "c5": 709123456,
+                "c6": 786452310,
+                "c7": None,
+                "c8": 274839201928,
+            },
+        ]
+
+
+class aggtst_varbinary_table(TstTable):
```

Comment:
I assume these are identical to the the previous ones, except they have "append_only"
I wonder whether we can simplify this by writing this as 
```
from tests.aggregate_tests.test_varbinary_tbl import aggtst_varbinary_table as VTBL;

def __init__(self):
   tbl = VTBL()
   self.sql = tbl.sql + "WITH ('append_only' = 'true')"
   self.data = tbl.data
```

Can you try to make a change just for one table and see if it works?
If it does, we can make the change for all tables.
This will make it easier to maintain this code in the future.

Reply:
You can also probably define the string `" WITH ('append_only' = 'true')"` as a global constant in `aggtst_base` (which is already imported)